src/_repobee/util.py

```diff
@@ -111,3 +112,14 @@ def call_if_defined(func: Callable[..., T], *args, **kwargs) -> T:
         What ``func`` returns, or ``None`` if ``func`` is ``None``.
     """
     return None if func is None else func(*args, **kwargs)
+
+
+def resolve_config_file(path: pathlib.Path,) -> pathlib.Path:
```

Comment:
this was moved into this file from `main.py` to avoid a circular dependency problem of preparser -> main -> preparser.

Reply:
given other changes here, this was moved back to being a private function in `main.py` so no change necessary.